content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -1,7 +1,136 @@
+
+Here's an example `.yaml` file that shows the required fields and object spec for a Kubernetes Deployment:
+
+{{< codenew file="application/deployment.yaml" >}}
+
+One way to create a Deployment using a `.yaml` file like the one above is to use the
+[`kubectl apply`](/docs/reference/generated/kubectl/kubectl-commands#apply) command
+in the `kubectl` command-line interface, passing the `.yaml` file as an argument. Here's an example:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/deployment.yaml
+```
+
+The output is similar to this:
+
+```
+deployment.apps/nginx-deployment created
+```
+
+### Required fields
+
+In the `.yaml` file for the Kubernetes object you want to create, you'll need to set values for the following fields:
+
+* `apiVersion` - Which version of the Kubernetes API you're using to create this object
+* `kind` - What kind of object you want to create
+* `metadata` - Data that helps uniquely identify the object, including a `name` string, `UID`, and optional `namespace`
+* `spec` - What state you desire for the object
+
+The precise format of the object `spec` is different for every Kubernetes object, and contains
+nested fields specific to that object. The [Kubernetes API Reference](/docs/reference/kubernetes-api/)
+can help you find the spec format for all of the objects you can create using Kubernetes.
+
+For example, see the [`spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
+for the Pod API reference.
+For each Pod, the `.spec` field specifies the pod and its desired state (such as the container image name for
+each container within that pod).
+Another example of an object specification is the
+[`spec` field](/docs/reference/kubernetes-api/workload-resources/stateful-set-v1/#StatefulSetSpec)
+for the StatefulSet API. For StatefulSet, the `.spec` field specifies the StatefulSet and
+its desired state.
+Within the `.spec` of a StatefulSet is a [template](/docs/concepts/workloads/pods/#pod-templates)
+for Pod objects. That template describes Pods that the StatefulSet controller will create in order to
+satisfy the StatefulSet specification.
+Different kinds of object can also have different `.status`; again, the API reference pages
+detail the structure of that `.status` field, and its content for each different type of object.
+
+## {{% heading "whatsnext" %}}
+
+If you're new to Kubernetes, read more about the following:
+
+* [Pods](/docs/concepts/workloads/pods/) which are the most important basic Kubernetes objects.
+* [Deployment](/docs/concepts/workloads/controllers/deployment/) objects.
+* [Controllers](/docs/concepts/architecture/controller/) in Kubernetes.
+* [kubectl](/docs/reference/kubectl/) and [kubectl commands](/docs/reference/generated/kubectl/kubectl-commands).
+
+To learn about the Kubernetes API in general, visit:
+
+* [Kubernetes API overview](/docs/reference/using-api/)
+
+To learn about objects in Kubernetes in more depth, read other pages in this section:
```

Comment:
(Docsy fills this bit in)

Reply:
The auto-filled index is terrible. Maybe we want to add a list manually?
Maintenance may not be a problem because this subdirectory is more stable than others.

Reply:
Can you explain what you don't like about it @tengqm? I think this page previews OK (it uses simple list mode).